src/sympc/tensor/static.py

```diff
@@ -0,0 +1,62 @@
+# third party
+import torch
+
+from sympc.utils import parallel_execution
+
+from .mpc_tensor import *  # why doesn't from .mpc_tensor import MPCTensor work?
```

Comment:
That is due to [circular/cyclic importing](https://stackoverflow.com/a/746067/8878627)

Reply:
Yep, thanks also noticed that still didn't work for me though. (the remote call).